### PR TITLE
feat(theme): added font size options

### PR DESCRIPTION
**Pull request by janhartmann**
This PR enables the control of the font sizes. We currently need this as part of a project and instead of maintaining a fork, we decided to push this change here.

### CODE REVIEW

File: src/flatTheme.ts [hunk 1]
```diff
@@ -66,6 +65,13 @@ export type FlatThemeOptions = {
      */
     fontFamily: string
 
+    fontSize?: {
+        chartTitle: number
+        axisTitle: number
+        lengendTitle: number
```
Comment (niiloArction) [inline src/flatTheme.ts:71]: typo, should be `legendTitle`

**Review by niiloArction — CHANGES_REQUESTED** [review]
We can merge these changes in to the main branch so they are included in next version release.

Please fix pending small issue before that though.

Thanks for contributing!

**Comment by janhartmann** [comment]
Oh, yes, was a mistake. Should be fixed now.

**Comment by niiloArction** [comment]
Would have also been ideal to mention the new capabilities in changelog https://github.com/Arction/lcjs-themes/blob/main/CHANGELOG.md

but I can quickly take care of this.

**Comment by janhartmann** [comment]
I will do that next time.

**Pull request merged**